targets/ath79/profiles/default/config

```diff
@@ -357,12 +358,17 @@ CONFIG_TARGET_MULTI_PROFILE=y
 # CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_belkin_f9k1115-v2 is not set
 # CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_bhr-4grv is not set
 # CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_bhr-4grv2 is not set
-# CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_wzr-600dhp is not set
-# CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_wzr-hp-ag300h is not set
-# CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_wzr-hp-g300nh-rb is not set
-# CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_wzr-hp-g300nh-s is not set
+CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_wzr-600dhp=y
+CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_buffalo_wzr-600dhp=""
+CONFIG_TARGET_DEVICE_ath79_generic_DEVICE_buffalo_wzr-hp-ag300h=y
+CONFIG_TARGET_DEVICE_PACKAGES_ath79_generic_DEVICE_buffalo_wzr-hp-ag300h="plugin-gargoyle-ddns plugin-gargoyle-ipheth-tether plugin-gargoyle-minidlna plugin-gargoyle-openvpn plugin-gargoyle-qr-code plugin-gargoyle-stamgr plugin-gargoyle-tor plugin-gargoyle-upnp plugin-gargoyle-usb-printer plugin-gargoyle-usb-storage-full plugin-gargoyle-webcam plugin-gargoyle-wireguard"
```

Comment:
Sorry, lines like this just kill me and I don't think this is the type of overhead that should need to be maintained in the repo.
Let me try a few things that might make this more acceptable and get back to you.